src/Data/Quantity/Math.purs

```diff
@@ -59,12 +71,21 @@ lift2 f q1 q2 = do
 acos ∷ Quantity → Result
 acos = lift Decimal.acos
 
+asec ∷ Quantity → Result
+asec = lift \x -> Decimal.acos (one / x)
+
 asin ∷ Quantity → Result
 asin = lift Decimal.asin
 
+acsc ∷ Quantity → Result
+acsc = lift \x -> Decimal.asin (one / x)
+
 atan ∷ Quantity → Result
 atan = lift Decimal.atan
 
+acot ∷ Quantity → Result
+acot = lift \x -> Decimal.atan (one / x)
+
 atan2 ∷ Quantity → Quantity → Result
 atan2 x y = removeDims <$> lift2 Decimal.atan2 x y
```

Comment:
To be frank, I don't really understand what `atan2` is; does it have a `cot` counterpart?

Reply:
Think of the following use case that is extremely common: if you want to go from polar coordinates r,phi to Cartesian coordinates x,y, you can simply use:

x = r * cos(phi)
y = r * sin(phi)

If you want go back, you can easily get r via

r = sqrt(x^2 + y^2)

However, it's not completely straightforward to get the phi angle from the values of x and y. We have

y/x = sin(phi)/cos(phi) * x = tan(phi)

which could be solved as phi = atan(y/x), but that is only true if x > 0. The function atan2(y, x) is basically a replacement for atan(y/x) for cases where x > 0, but in addition, it also handles all other (special) cases just fine and you can get the angle phi back via

phi = atan2(y, x)


> does it have a cot counterpart?

I'm not aware of it. Might be because it's not needed or simply because cotangent is used much less frequently than tangent.

Reply:
Thanks, now I know what `atan2` is.

> I'm not aware of it. Might be because it's not needed or simply because cotangent is used much less frequently than tangent.

Then I guess all that remains is adding tests.

Reply:
> Then I guess all that remains is adding tests.

That would be great. Because we do not just import these functions from decimal.js. Adding one or two values (maybe one negative, one positive) would be great. We can use Wolfram Alpha or Wikipedia as a source of the expected values.

Reply:
Sounds good. I also imagined some sort of test that checks each function against its inverse to ensure they really are inverses.